### PR TITLE
convert S3 to cdk-runner

### PR DESCRIPTION
**Pull request by sosodev**

### CODE REVIEW

File: s3/Acornfile [hunk 1]
```diff
@@ -7,9 +7,34 @@ args: {
 	tags: {}
 }
 
-services: {
-	"s3-bucket": generated: job: "apply"
-	"cdk-context": build: "../utils/cdk"
+services: "s3-bucket-read": {
```
Comment (cloudnautique) [inline s3/Acornfile:10]: Based on the SQS discussions, we should shorten the names to just the function and add one more:
`readonly`
lets add this one and make it the default.
`readwrite` -> Make the default
`writeonly`
`admin`

**Review by cloudnautique — COMMENTED** [review]
Overall looking good.

File: s3/Acornfile (hunk 1 above)
Comment (sosodev) [reply s3/Acornfile:10]: Should deletion be included along with the write permissions or is that reserved for admin usage?

File: s3/Dockerfile [hunk 2]
```diff
@@ -2,26 +2,21 @@ FROM cgr.dev/chainguard/go as build
...
 COPY . .
 RUN --mount=type=cache,target=/root/go/pkg \
     --mount=type=cache,target=/root/.cache/go-build \
     ls ../ &&\
-    go build -o s3 . && \
-    cd ../cfn-events && \
-    go build -o cfn-events .
+    go build -o s3 .
 
+FROM ghcr.io/acorn-io/aws/utils/cdk-runner:v0.2.0 as cdk-runner
```
Comment (cloudnautique) [inline s3/Dockerfile:11]: Can you bump to v0.3.0

**Review by cloudnautique — CHANGES_REQUESTED** [review]
Just need to bump cdk-runner.

File: s3/Dockerfile (hunk 2 above)
Comment (sosodev) [reply s3/Dockerfile]: done

**Review by cloudnautique — APPROVED** [review]

**Pull request merged**